roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - pip
+  - logrotate
```

Comment:
What jamie is saying makes sense to me. We should probably avoid unnecessary dependencies if we can help it.